src/main/java/org/kainos/ea/validator/JobCapabilityValidator.java

```diff
@@ -0,0 +1,14 @@
+package org.kainos.ea.validator;
+
+import org.kainos.ea.cli.CapabilityRequest;
+import org.kainos.ea.client.CapabilityNameTooLongException;
+
+public class JobCapabilityValidator {
+    public boolean isValidCapability(CapabilityRequest capabilityRequest) throws CapabilityNameTooLongException {
```

Comment:
Are you using the validator anywhere

Reply:
Validator is being used in the JobCapabilityController class